app/views/posts/edit.html.erb

```diff
@@ -1,3 +1,14 @@
 <h1>EDIT</h1>
 <hr>
-<%= render "form" %>
+<%= form_for @post, url: team_post_path, html: { method: :put } do |f| %>
```

Comment:
I don't think you need to specify that method here when the post is already filled it wil make it an edit